pages.sr/common/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Menja trnutnu radnu površinu..
+> Više informacija na: <https://man.archlinux.org/man/cd.n>.
+
+- Ulazi u dati direktorijum:
+
+`cd {{path/to/directory}}`
```

Comment:
Please translate the arguments for get this easier for non-English speakers. Could this `path/to/directory` be translated?

Reply:
I wasn't sure about touching arguments, if you say it's okay I think I could translate it in such a way that it keeps it's meaning.